packages/abi-typegen/src/types/B256Type.ts

```diff
@@ -0,0 +1,13 @@
+import { StrType } from './StrType';
+
+export class B256Type extends StrType {
+  public static swayTypeExample = 'b256';
+
+  public name = 'b256';
+
+  static MATCH_REGEX = /^b256$/m;
```

Comment:
I didn't understand why having the `swayTypeExample`.

It is just for docs we could add just as docs. I saw it had been used just on `tests`.



```suggestion
  public static swayTypeExample = 'b256';

  public name = 'b256';

  static MATCH_REGEX = /^b256$/m;
```